### DISCUSSION TITLE
如何使用自定义字体？

### DISCUSSION THREAD

**Discussion by xixihhhh** [openkraken/kraken#170] [Q&A]
看文档好像没有font-face，那要怎么导入字体呢

**Comment by andycall** [comment]
现在还不支持 font-face 这样的 CSS 语法，所以不支持动态加载字体的功能。
不过可以预先将字体打包到 App 中，然后使用 font-family 设置字体，可以参考这里的用法

https://github.com/openkraken/kraken/blob/main/integration_tests/pubspec.yaml#L69
